spec/models/work_packages/pdf_export/work_package_to_pdf_spec.rb

```diff
@@ -71,6 +71,10 @@
            # cf_disabled_in_project.id not included == disabled
            work_package_custom_field_ids: [cf_long_text.id, cf_empty_long_text.id, cf_global_bool.id, cf_link.id])
   end
+  let(:phase_definition) { create(:project_phase_definition, name: "Test Phase") }
+  let!(:project_phase) do
+    create(:project_phase, project: project, definition: phase_definition, active: true)
+  end
```

Comment:
No need to explicitly create phase definition, it will be built/created automatically
```suggestion
  let!(:project_phase) do
    create(:project_phase, project: project, active: true)
  end
```

Reply:
removed https://github.com/opf/openproject/pull/19449/commits/2d414f6d387e08db966f00d54a29c946ee205ae2